### PR TITLE
Formatting and linting

### LINKED ISSUE #98: Format the code
# Description
Once @marcomangano finishes the PR on rotating frames, the python code needs to be formatted and all linting errors fixed.

### PR DESCRIPTION
**Pull request by sseraj**
## Purpose
Closes #98

Please review the linting changes carefully and comment if it's unclear why I made a certain change.

For flake8, I ignored 
```
src_cs/build/complexify.py
adflow/pyWeightAndBalance.py
adflow/pyWingCG.py
```

The latter two are extremely outdated and should probably be removed from the repo. I can remove them in this PR or we can address that later. Similarly, we may also want to remove pyDynamicPerformance.py and pyPerformanceClass.py.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

## Testing
Explain the steps needed to test the new code to verify that it does indeed address the issue and produce the expected behavior.

## Checklist
_Put an `x` in the boxes that apply._

- [x] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [x] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: adflow/pyADflow.py [hunk 1]
```diff
@@ -2085,14 +2085,15 @@ def writeSolution(self, outputDir=None, baseName=None, number=None):
...
-
-        outputDir: Use the supplied output directory
+        Parameters
+        ----------
 
-        baseName: Use this supplied string for the base filename. Typically
-                  only used from an external solver.
-        number: Use the user supplied number to index solution. Again, only
-                typically used from an external solver.
+        outputDir: str
```
Comment (nwu63) [inline adflow/pyADflow.py:2091]: Space before the colon?

**Comment by nwu63** [comment]
That was fast! Do you want to add a `ignore-revs` file as we did for other repos?

File: adflow/pyADflow.py [hunk 2]
```diff
@@ -2507,7 +2498,7 @@ def resetFlow(self, aeroProblem, releaseAdjointMemory=True):
         # time this AP is used.
         try:
             aeroProblem.adflowData.oldWinf = None
-        except:
+        except Exception:
```
[the remark this answers is not in the document]
Comment (sseraj) [reply adflow/pyADflow.py:2502]: I think for `pass` statements I just used Exception but might as well be more specific.

[the remark this answers is not in the document]
Comment (nwu63) [reply adflow/pyADflow.py]: Yeah not a big deal, this is not high quality code anyways (it shouldn't just inject these fields and then throw AttributeError if they don't exist, that's definitely not the proper way).

**Review by nwu63 — DISMISSED** [review]

**Comment by anilyil** [comment]
The changes look good, I dont know what more I can add here because all tests pass. @sseraj dont forget to bump the version number. Are there specific things you want me to look at?

**Comment by sseraj** [comment]
> The changes look good, I dont know what more I can add here because all tests pass. @sseraj dont forget to bump the version number. Are there specific things you want me to look at?

The [linting changes](https://github.com/mdolab/adflow/pull/135/commits/68d39bc64183234b3853d372828f7a1354df6380) are really the only part I wanted a review of, in case I broke something not covered by the tests. If those look fine to you, then I think this is good to merge.

**Review by anilyil — APPROVED** [review]
I took a look over the linting changes, and they look good to me. Thanks for working on this!

**Review by nwu63 — APPROVED** [review]

**Comment by nwu63** [comment]
Don't forget to make a release @sseraj. Thanks for working on this!

**Pull request merged**